realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -245,15 +263,7 @@ public static void validateTable(ImplicitTransaction transaction) {
                 }
                 columnIndices.put(fieldName, index);
             }
-            INDEX_COLUMNSTRING = table.getColumnIndex("columnString");
-            INDEX_COLUMNLONG = table.getColumnIndex("columnLong");
-            INDEX_COLUMNFLOAT = table.getColumnIndex("columnFloat");
-            INDEX_COLUMNDOUBLE = table.getColumnIndex("columnDouble");
-            INDEX_COLUMNBOOLEAN = table.getColumnIndex("columnBoolean");
-            INDEX_COLUMNDATE = table.getColumnIndex("columnDate");
-            INDEX_COLUMNBINARY = table.getColumnIndex("columnBinary");
-            INDEX_COLUMNOBJECT = table.getColumnIndex("columnObject");
-            INDEX_COLUMNREALMLIST = table.getColumnIndex("columnRealmList");
+            final AllTypesColumnInfo columnInfo = new AllTypesColumnInfo(columnIndices, table.getColumnIndex("columnString"), table.getColumnIndex("columnLong"), table.getColumnIndex("columnFloat"), table.getColumnIndex("columnDouble"), table.getColumnIndex("columnBoolean"), table.getColumnIndex("columnDate"), table.getColumnIndex("columnBinary"), table.getColumnIndex("columnObject"), table.getColumnIndex("columnRealmList"));
```

Comment:
Is it more performant to do this in the constructor? I preferred the old formatting as it was way more readable. The variables could just be local.